server.js

```diff
@@ -620,10 +620,30 @@ function getConfigData(req) {
     }/auth/realms/openshift/protocol/openid-connect/logout?redirect_uri=${logoutRedirectUri}',
     threescaleWildcardDomain: '${process.env.THREESCALE_WILDCARD_DOMAIN || ''}',
     integreatlyVersion: '${process.env.INTEGREATLY_VERSION || ''}',
-    clusterType: '${process.env.CLUSTER_TYPE || ''}'
+    clusterType: '${process.env.CLUSTER_TYPE || ''}',
+    optionalWatchServices: ${arrayFromString(process.env.OPTIONAL_WATCH_SERVICES, ',')},
+    optionalProvisionServices: ${arrayFromString(process.env.OPTIONAL_PROVISION_SERVICES, ',')}
```

Comment:
both those vars should hold a comma separated value of additonal services - I decided to use two arrays so it would be easiet to set it in ansible